.openpublishing.redirection.csharp.json

```diff
@@ -4052,6 +4052,10 @@
             "source_path_from_root": "/docs/csharp/roslyn/index.md",
             "redirect_url": "/dotnet/csharp/roslyn-sdk/"
         },
+        {
+            "source_path_from_root": "/docs/csharp/roslyn-sdk/source-generators-overview.md",
+            "redirect_url": "/dotnet/csharp/roslyn-sdk/#source-generators"
```

Comment:
This bookmark doesn't seem to exist. And since you changed some source generator links from e.g. STJ to point here, I think that's a problem.

Reply:
It's added in this PR. I tested on the review site, and it works for me.